ebs_snapshots/snapshot_manager.py

```diff
@@ -96,25 +98,53 @@ def _copy_snapshot(backup_client, volume, snapshot_id, name):
     """
     logging.info(kayvee.formatLog("ebs-snapshots", "info", "copying snapshot", {"volume": volume.id, "source_snapshot": snapshot_id}))
     region = _availability_zone_to_region_name(volume.zone)
-    response = backup_client.copy_snapshot(
-        SourceRegion=region,
-        SourceSnapshotId=snapshot_id,
-        Encrypted=True,
-        Description='copy of {}'.format(snapshot_id))
-
-    backup_client.create_tags(
-        Resources=[response["SnapshotId"]],
-        Tags=[
-            {"Key":"Name", 'Value':name},
-            {"Key":"creator", 'Value':"ebs-snapshots"},
-            {"Key":"snapshot_source", 'Value':snapshot_id},
-            {"Key":"volume-id", 'Value':volume.id}
-        ]
-    )
+
+    try:
+        response = backup_client.copy_snapshot(
+            SourceRegion=region,
+            SourceSnapshotId=snapshot_id,
+            Encrypted=True,
+            Description='copy of {}'.format(snapshot_id))
+
+    except ClientError as error:
+        if error.response["Error"]["Code"] == "ResourceLimitExceeded":
+            logging.info(kayvee.formatLog("ebs-snapshots", "info", "copying snapshot", {
+                "volume": volume.id,
+                "source_snapshot": snapshot_id,
+                "name": name
+            }))
```

Comment:
If I'm reading this correctly, the worker now will run, initiate 5 snapshot copies, and then hit the parallel copy limit on snapshot numbers 6, 7, etc. So there's a world where every run of the worker results in only the first five snapshots get copied, and the rest fail with this error log... is that correct?

Reply:
Yes, we will see this particular error pretty frequently as we build up the 1512 snapshots specified in [ark-config](https://github.com/Clever/ark-config/blob/master/apps/ebs-snapshots/production.yml).   We can do 5 at a time, but there's an hour cooldown between taking snapshots for the same volume.  I think it'll stabilize within a few weeks.